src/lua/fio.lua

```diff
@@ -99,13 +99,21 @@ end
 -- pread(size, offset) -> str
 -- pread(buf, size, offset) -> len
 fio_methods.pread = function(self, buf, size, offset)
-    local tmpbuf
+    local no_buf = false
     if not ffi.istype(const_char_ptr_t, buf) then
         offset = size
         size = buf
+        no_buf = true
+    end
+    if (type(size) ~= 'number' or (type(offset) ~= 'number' and offset ~= nil)) then
+        error('usage: fh:pread([buf,] size, offset = 0)')
```

Comment:
Nit: Python style named parameter syntax may be confusing in the Lua context (is it positional or a table is assumed around?). I propose the following:

```lua
error('Usage: fh:pread(buf, size[, offset]) or fh:pread(size[, offset])')
```

Reply:
Fixed.